ydb/docs/en/core/concepts/backup-collections.md

```diff
@@ -0,0 +1,308 @@
+
+## What are backup collections? {#what-are-backup-collections}
+
+A backup collection is a named set of coordinated backups for selected database objects. Collections organize related backups and ensure they can be restored together consistently, providing:
+
+- **Organization** — related backups are grouped into logical collections.
+- **Recovery flexibility** — enables recovery to any point in time when one of the backups in the collection was made.
+
+{% note info %}
+
+Currently only row-oriented tables are supported.
+
+{% endnote %}
+
+## Core concepts {#core-concepts}
+
+### Full backup {#full-backup}
+
+A snapshot at a specific point in time. Serves as the foundation for subsequent incremental backups and contains all data necessary for independent restoration.
+
+### Incremental backup {#incremental-backup}
+
+Captures only changes (inserts, updates, deletes) since the previous backup in the chain. Significantly smaller than full backups with low change frequency.
+
+### Backup chain {#backup-chain}
+
+An ordered sequence of backups starting with a full backup followed by zero or more incremental backups. Each incremental backup depends on all previous backups in the chain.
+
+## Backup workflow {#backup-workflow}
+
+```mermaid
+block-beta
+    columns 11
+    space:11
+    CREATE["CREATE<br/>COLLECTION"]
+    space
+    FULL["BACKUP<br/>FULL<br/>100%"]
+    space
+    INC1["BACKUP<br/>INCREMENTAL<br/>+5%"]
+    space
+    INC2["BACKUP<br/>INCREMENTAL<br/>+5%"]
+    space
+    INC3["BACKUP<br/>INCREMENTAL<br/>+5%"]
+    space
+    DROP["DROP<br/>COLLECTION"]
+    
+    space:11
+    
+    space:2
+    FS1["Filesystem<br/>Export"]
+    space
+    FS2["Filesystem<br/>Export"]
+    space
+    FS3["Filesystem<br/>Export"]
+    space
+    FS4["Filesystem<br/>Export"]
+    
+    CREATE --> FULL
+    FULL --> INC1
+    INC1 --> INC2
+    INC2 --> INC3
+    INC3 --> DROP
+    
+    FULL -.-> FS1
+    INC1 -.-> FS2
+    INC2 -.-> FS3
+    INC3 -.-> FS4
+    
+    style CREATE fill:#E3F2FD
+    style FULL fill:#2196F3,color:#fff
+    style INC1 fill:#90CAF9
+    style INC2 fill:#BBDEFB
+    style INC3 fill:#E3F2FD
+    style DROP fill:#FFB6C1
+    style FS1 fill:#FFF9C4
+    style FS2 fill:#FFF9C4
+    style FS3 fill:#FFF9C4
+    style FS4 fill:#FFF9C4
+```
+
+### Collection creation {#collection-creation}
+
+Creates a named container that coordinates backups across multiple database objects. All objects in the collection share the same backup configuration and storage location. Once created, the object list is fixed and cannot be modified.
+
+[See operations guide](../maintenance/manual/backup-collections.md#create-collection)
+
+### Initial full backup {#initial-full-backup}
+
+Creates a consistent snapshot of all objects in the collection at a specific point in time. This baseline backup contains all data and can be restored independently without requiring other backups. A full backup is required before creating any incremental backups.
```

Comment:
Explain what guarantees to expect from this. For instance, is it executed against a global snapshot that ensures referential integrity across objects?